test/src/main/java/org/axonframework/test/saga/SagaTestFixture.java

```diff
@@ -145,15 +149,9 @@ protected void handleInSaga(EventMessage<?> event) {
      * @param sagaDescriptor  A {@link ScopeDescriptor} describing the saga under test
      * @param deadlineMessage The {@link DeadlineMessage} to be handled
      */
-    protected void handleDeadline(ScopeDescriptor sagaDescriptor, DeadlineMessage<?> deadlineMessage) {
+    protected void handleDeadline(ScopeDescriptor sagaDescriptor, DeadlineMessage<?> deadlineMessage) throws Exception {
         ensureSagaResourcesInitialized();
-        DefaultUnitOfWork.startAndGet(deadlineMessage).execute(() -> {
```

Comment:
Similar question as with the Aggregate fixture stuff: how com you can drop the unit of work altogether here?